mmv1/templates/terraform/custom_check_destroy/iam_workforce_pool.go.erb

```diff
@@ -0,0 +1,17 @@
+config := googleProviderConfig(t)
+
+url, err := replaceVarsForTest(config, rs, "{{IAMBetaBasePath}}locations/{{location}}/workforcePools/{{workforce_pool_id}}")
+if err != nil {
+  return err
+}
+
+res, err := sendRequest(config, "GET", "", url, config.userAgent, nil)
+if err != nil {
+  return nil
+}
+
+if v := res["state"]; v == "DELETED" {
+	return nil
+}
+
+return fmt.Errorf("IAMWorkforcePool still exists at %s", url)
```

Comment:
Can you add trailing newlines to all these files? It will match our existing formatting better

Reply:
fixed